cmd/devstream/main.go

```diff
@@ -43,6 +44,7 @@ func init() {
 	rootCMD.PersistentFlags().StringVarP(&pluginDir, "plugin-dir", "p", pluginengine.DefaultPluginDir, "plugins directory")
 	rootCMD.PersistentFlags().BoolVarP(&continueDirectly, "yes", "y", false, "apply/delete directly without confirmation")
 	rootCMD.PersistentFlags().BoolVarP(&isDebug, "debug", "d", false, "debug level log")
+	rootCMD.PersistentFlags().BoolVarP(&isForce, "force", "r", false, "force to delete from config")
```

Comment:
"isForce" should not belong to "rootCMD". It should belong to "delete" subcommand only, because we won't have "dtm apply --force".